src/main/kotlin/io/github/enyason/predictions/models/PaginatedPredictionsDTO.kt

```diff
@@ -0,0 +1,7 @@
+package io.github.enyason.predictions.models
+
+data class PaginatedPredictionsDTO(
+    val next: String?,
+    val previous: String?,
+    val results: List<PredictionDTO<Any>>
```

Comment:
Can this be null? In a situation where a user has never created any prediction, will this be null or empty?

Reply:
It returns an empty list